coding-concepts.js

```diff
@@ -67,7 +67,7 @@ class Learn {
   }
 }
 var learnStudent = new Learn("George")
-// console.log(learnStudent)
+console.log(learnStudent)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: I think it should return an object with the following key value pairs: student: "George", cohort: "Delta", year: 2021
+// b) Verify and explain: It returned the object
```

Comment:
Just a note that the instructions for the second part of each answer are: `Briefly explain why your initial answer was correct or incorrect.`